src/libs/Navigation/AppNavigator/DrawerNavigator/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import withWindowDimensions from '../../../../components/withWindowDimensions';
+import BaseDrawerNavigator from './BaseDrawerNavigator';
+import propTypes from './drawerNavigatorPropTypes';
+
+const DrawerNavigator = props => (
+    <BaseDrawerNavigator
+        // eslint-disable-next-line react/jsx-props-no-spreading
+        {...props}
+
+        // On mobile web, the non legacy version of the DrawerNavigator breaks the layout on Desktop while resizing
+        // and the drawer gets stuck in an open state when navigating.
```

Comment:
What about this?

```suggestion
        // On mobile web, the non legacy (modern) version of the DrawerNavigator has issues opening and closing
        // the drawer, so we always use the legacy version.
        // On web & desktop, the legacy version is needed on small screen sizes and modern elsewhere, otherwise
        // the layout breaks when the screen is resized.
```

Reply:
Sorry, it seems, too wordy. Maybe we can shorten it a bit.

Reply:
On the mobile web, the non-legacy (modern) version of the DrawerNavigator has issues opening and closing, and the legacy version breaks the layout while resizing on Desktop.

Reply:
If you think this is fine, I will commit. Otherwise, let's go with yours. @Beamanator

Reply:
I like it 👍 But at the end, please add "and web" so we know "the legacy version breaks the layout while resizing on desktop AND web"